EasyPost/BetaFeatures/Parameters/Shipments/All.cs

```diff
@@ -42,6 +42,18 @@ public sealed class All : BaseParameters
         [TopLevelRequestParameter(Necessity.Optional, "start_datetime")]
         public string? StartDatetime { get; set; }
 
+        /// <summary>
+        ///     If true, only return purchased shipments. Defaults to true.
+        /// </summary>
+        [TopLevelRequestParameter(Necessity.Optional, "purchased")]
+        public bool? Purchased { get; set; } = true;
+
+        /// <summary>
+        ///     If true, also include shipments created by child users.
+        /// </summary>
+        [TopLevelRequestParameter(Necessity.Optional, "include_children")]
+        public bool? IncludeChildren { get; set; }
```

Comment:
```suggestion
        public bool? IncludeChildren { get; set; } = false;
```

Question: should we also include default value for `IncludeChildren`

Reply:
Thoughts @Justintime50 ? We could always send `false` by default. Or, if we don't indicate a value here, it's wired to not send anything, which means it would fall back to the server (which, according to our docs, defaults to `false` for this).

I believe we spoke earlier about wanting all boolean parameters, other than carbon offset, to not be included unless explicitly set by the end-user.

Reply:
I asked this because there is a default value for `purchased` in the above lines.

Reply:
Let's try not to set these and let the API handle the defaults. If ever they changed, we wouldn't need to make alterations to the libs when they aren't set.